tests/client/gridfs.rs

```diff
@@ -34,12 +34,12 @@ fn put_get() {
     let (fs, _, fschunks) = init_gridfs("test-client-gridfs-grid_put");
 
     let name = "grid_put_file";
-    let src_len = (DEFAULT_CHUNK_SIZE as f32 * 2.5) as usize;
+    let src_len = (DEFAULT_CHUNK_SIZE as f64 * 2.5) as usize;
```

Comment:
This is an interesting one. I guess it makes sense to use f64 if there's not a reason not to to avoid losing precision, but it doesn't really matter here that much. (If anything,  `DEFAULT_CHUNK_SIZE * 5 / 2` might even be a bit more clean, although I'd want to verify that it doesn't overflow since DEFAULT_CHUNK_SIZE is an i32).